### ISSUE TITLE
Can't set installation directory

### ISSUE THREAD

**Issue by eco9898** [RLBot/RLBotGUI#186]
I am using epic games to play rocket league. I changed my drive's letter a few months back and RLBotGui is trying to launch RL by the old drive letter, epic games uses the correct drive letter but RLBotGui won't detect the correct installation. I can't find any option to manually set the install directory and didn't find any help in the RLBot discord or this github.

An option to set the directory would be helpful to resolve this issue. If it already exists could someone let me know how to access it?

**Comment by eco9898** [comment]
manually starting the game with the arguments via cmdline or epic games settings causes it to launch but not work correctly. online play is working properly, which it shouldn't  be? Also, RLBot won't start a match:
`2022-04-30 08:39:19,317 INFO:rlbot[    setup_manager.py:400 - ensure_rlbot_gateway_started() ] Already have RLBot.exe running! Port is 23233
2022-04-30 08:39:20,797 INFO:rlbot[    setup_manager.py:211 -      connect_to_game() ] Loading interface...
2022-04-30 08:39:20,998 INFO:rlbot[   game_interface.py:264 -    wait_until_loaded() ] DLL is initialized!
2022-04-30 08:39:20,999 INFO:rlbot[   game_interface.py:338 -       load_interface() ] About to call StartTcp`

**Comment by eco9898** [comment]
I found the code is searching `C:\ProgramData\Epic\EpicGamesLauncher\Data\Manifests` for a manifest file and using that directory so I will try to modify the manifests there. Still, there should be a way to set the location with a manual overide

**Comment by tarehart** [comment]
You're right, we should make a way to set an override. Thanks for documenting the workaround in the meantime

**Comment by kaiserbleacher** [comment]
Buenas, quería saber si hay una forma de poder jugar rocket league con RLBot sin necesidad de correr epic games o steam. Yo tengo el cliente del juego completo pero cuando le pongo la ubicación del ejecutable del juego, se me inicia el juego cuando toco el botón de empezar partido, pero no se inicia ningún partido con los bots seleccionados
